Description: Fix LDFLAGS usage, 
Author: Gianfranco Costamagna <locutusofborg@debian.org>

Forwarded: https://github.com/apache/trafficserver/pull/4793
Last-Update: 2019-01-12

--- trafficserver-8.0.1.orig/build/common.m4
+++ trafficserver-8.0.1/build/common.m4
@@ -196,17 +196,17 @@ AC_DEFUN([TS_TRY_COMPILE_NO_WARNING],
 ])
 
 dnl
-dnl TS_LINK_WITH_FLAGS_IFELSE(LDFLAGS, FUNCTION-BODY,
+dnl TS_LINK_WITH_FLAGS_IFELSE(LIBS, FUNCTION-BODY,
 dnl                           [ACTIONS-IF-LINKS], [ACTIONS-IF-LINK-FAILS])
 dnl
 dnl Tries a link test with the provided flags.
 dnl
 
 AC_DEFUN([TS_LINK_WITH_FLAGS_IFELSE],
-[ats_save_LDFLAGS=$LDFLAGS
- LDFLAGS="$LDFLAGS $1"
+[ats_save_LIBS=$LIBS
+ LIBS="$LIBS $1"
  AC_LINK_IFELSE([$2],[$3],[$4])
- LDFLAGS=$ats_save_LDFLAGS
+ LIBS=$ats_save_LIBS
 ])
 
 
